[SYCL] Support memory clock rate and memory bus width queries (#7412)
intel/llvm: sycl/doc/extensions/supported/sycl_ext_intel_device_info.md

| 3     | HW threads per EU device query is supported |
| 4     | Free device memory query is supported |
| 5     | Device ID is supported |

> | 6     | Memory clock rate and bus width queries are supported |

Author: Thanks for catching this, fixed.

      auto FreeMemory = dev.get_info<ext::intel::info::device::free_memory>();
    }

# Memory Clock Rate #

> A new device descriptor will be added which will provide the maximum clock rate of device's global memory.

Reviewer: ```suggestion
A new device descriptor is added which provides the maximum clock rate of device's global memory.
```
?

Author: Fixed.
